lib/linter.js

```diff
@@ -71,8 +71,8 @@ linter.lintFiles = function (files, standardOptions, done) {
   }
   async.map(files, function (file, callback) {
     // Inform the user of progress
-    process.stdout.clearLine()
-    process.stdout.cursorTo(0)
+    process.stdout.clearLine && process.stdout.clearLine()
```

Comment:
Although `standard` allows short circuiting like this I find it makes code harder to read and understand what it is doing.

Something like this makes the code easier to understand / read I feel
```js
if (process.stdout.clearLine && process.stdout.cursorTo) {
  process.stdout.clearLine()
  process.stdout.cursorTo(0)
}
process.stdout.write('Linting file ' + (index++ + 1) + ' of ' + files.length)
```